src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java

```diff
@@ -36,20 +35,21 @@ public class GenomicEntity extends BiologicalEntity {
 
 	@IndexedEmbedded(includePaths = {"referencedCurie", "displayName", "referencedCurie_keyword", "displayName_keyword"})
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-	@ManyToMany
+	@OneToMany(cascade = CascadeType.ALL, orphanRemoval=true)
```

Comment:
We've probably had this conversation before, but I think this needs to be OneToMany for orphanRemoval to work - otherwise we need to manually delete xrefs when we delete a GenomicEntity